src/EditorFeatures/Test/StackTraceExplorer/StackTraceExplorerTests.cs

```diff
@@ -612,14 +650,102 @@ void LocalFunction()
 }");
         }
 
-        [Fact(Skip = "Generated types/methods are not supported")]
+        [Fact]
+        public Task TestSymbolFound_ExceptionLine_MultipleLocalFunctions2()
+        {
+            return TestSymbolFoundAsync(
+                @"at ConsoleApp4.MyClass.<M2>g__LocalFunction|0_0()",
+                @"using System;
+
+namespace ConsoleApp4
+{
+    class MyClass
+    {
+        public void M()
+        {
+            LocalFunction();
+
+            void LocalFunction()
+            {
+                throw new Exception();
+            }
+        }
+
+        public void M2()
+        {
+            LocalFunction();
+
+            void [|LocalFunction()|]
+            {
+                throw new Exception();
+            }
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_ExceptionLine_MemberFunctionSameNameAsFunction()
+        {
+            return TestSymbolFoundAsync(
+                @"at ConsoleApp4.MyClass.LocalFunction()",
+                @"using System;
+
+namespace ConsoleApp4
+{
+    class MyClass
+    {
+        public void M()
+        {
+            LocalFunction();
+
+            void LocalFunction()
+            {
+                throw new Exception();
+            }
+        }
+
+        public void [|LocalFunction|]()
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_ExceptionLine_NestedLocalFunctions()
```

Comment:
This test currently will fail. I haven't figured out the best way to go from the symbols for LocalFunctions to the generated slots.

Reply:
what's the issue here?

Reply:
oh, is it that the outer and inner local function names are teh same?  yeah, we may not be able to support hat.  do you have tests where the names are different?

Reply:
I'll add a test with different names. The issue will always be if there is something with the same name in the containing member. We could try to reverse engineer it but likely it would be wrong. 

I wonder if just going to the containing method in those cases would be a good compromise.